test/instrumentation/modules/aws-sdk/fixtures-dynamodb.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
```

Comment:
Thoughts on moving this fixture to the "fixtures/" subdir? Currently "test/test.js" will be attempting to execute this file as a test because it looks for all "*.js" files in this dir (and excludes those beginning with an underscore). The "fixtures/" subdir here is not on its list of dirs to look for test files.

Reply:
sure thing -- I'll move the SQS fixtures files while I'm in there.